README.md

```diff
@@ -1 +1,144 @@
-# leaderboard-app
\ No newline at end of file
+<div align="center">
+  <h3><b>Leader board App</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Leader board App ](#-leaderboard-app-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+    - [🚀 Live Demo](#live-demo)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [📝 License ](#-license-)
+
+# 📖 Leader board App <a name="leaderboard-app"></a>
+**Leader board App**  A JavaScript project for the Leaderboard list app, using webpack and ES6 features, notably modules. Developed a first working version of the app following a wireframe, but with minimal styling - just focus on functionality. Consumes the Leaderboard API using JavaScript async and await.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>Javascript</li>
+    <li>CSS</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Has a page with a list of scores**
+- **Has a section where a user can add a new score**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+-[Live Demo Link](https://veronica365.github.io/leaderboard-app/dist/)
+
```

Comment:
- Kindly add a valid demo link to your live demo, or add coming soon... if you don't have one yet 😉.